README.md

```diff
@@ -3,4 +3,5 @@ Demo repository for the tutorial about GitHub Pull Requests on [genicsblog.com](
 
 Please [read the tutorial](https://genicsblog.com/gouravkhunger/how-to-create-a-pull-request-in-github-correctly) before creating a pull request!
 
-- This line was added by [Gourav](https://github.com/gouravkhunger).
\ No newline at end of file
+- This line was added by [Gourav](https://github.com/gouravkhunger).
+- This line was added by [<Alexis>](https://github.com/alexisCode50).
```

Comment:
`<Alexis>` doesn't render a link properly. Please change `<Alexis>` to `Alexis` and it'll be fine!